src/simulations/results/results_plotting_utils.jl

```diff
@@ -0,0 +1,406 @@
+# Dictionary of variable-specific titles
+title_mapping = Dict(
```

Comment:
A very important thing in Julia: NEVER write code in the global scope (i.e. at the root of a file). ALWAYS put all code inside a function. If you need this title_mapping in all the plotting functions, you should think of putting this somewhere in a `struct` that is passed to the plotting functions. 

Perhaps the easiest way would be to have ONLY one plotting function that serves as a hub or input, where you initialize this dict, and then, depending on flags or variables that you pass at the input, this function calls other functions inside with the right plotting type. This would make things much more tidy.

Reply:
Fixed by creating hub function